File: GUI/output/output.ts
/** This is the `Output` class */
class Output {
  id: string;
  index: number;
  guid: string;
  isActive: boolean;
  balance: string;
  picture: string;
  age: number;
  eyeColor: string;
  name: string;
  gender: string;
  company: string;
  email: string;
  phone: string;
  addres: string;
  about: string;
  registered: string;
  latitude: number;
  longitude: number;
  tag: string[];
  friend: Friend[];
  greeting: string;
  favoriteFruit: string;

  constructor(json) {
    this.id = json["_id"];
    this.index = json["index"];
    this.guid = json["guid"];
    this.isActive = json["isActive"];
    this.balance = json["balance"];
    this.picture = json["picture"];
    this.age = json["age"];
    this.eyeColor = json["eyeColor"];
    this.name = json["name"];
    this.gender = json["gender"];
    this.company = json["company"];
    this.email = json["email"];
    this.phone = json["phone"];
    this.addres = json["address"];
    this.about = json["about"];
    this.registered = json["registered"];
    this.latitude = json["latitude"];
    this.longitude = json["longitude"];
    this.tag = json["tags"];
    this.friend = json["friends"];
    this.greeting = json["greeting"];
    this.favoriteFruit = json["favoriteFruit"];
  }
}

/** This is the `Friend` class */
class Friend {
  id: number;
  name: string;

  constructor(json) {
    this.id = json["id"];
    this.name = json["name"];
  }
}

File: GUI/output/output.kt
/** This is the `Output` class */
data class Output (
  val id: String,
  val index: Int,
  val guid: String,
  val isActive: Boolean,
  val balance: String,
  val picture: String,
  val age: Int,
  val eyeColor: String,
  val name: String,
  val gender: String,
  val company: String,
  val email: String,
  val phone: String,
  val addres: String,
  val about: String,
  val registered: String,
  val latitude: Double,
  val longitude: Double,
  val tag: List<String>,
  val friend: List<Friend>,
  val greeting: String,
  val favoriteFruit: String
)

/** This is the `Friend` class */
data class Friend (
  val id: Int,
  val name: String
)
